tests/test_ClusterCon_SentinelCon.py

```
@@ -0,0 +1,118 @@
+import pytest
+from unittest.mock import MagicMock, patch
```

Ruff

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)